test/test-template-engine.js

```diff
@@ -884,7 +884,7 @@ describe('rendering errors', () => {
       } catch (e) {
         error = e;
       }
-      assert(error?.message === `Found invalid template string "<div id="target" .notOk=" at " .notOk=".`, error.message);
+      assert(error?.message === `Found invalid template at line 4 within string \`\n\n\n<div id="target" .notOk=\` at \` .notOk=\`.`, error.message);
```

Comment:
The `at line 4` text here is wildly helpful . It makes it exceedingly easy to track down template issues. IMO, if we’re gonna be strict about templates, we should strive to be increasingly specific about what went wrong and where.